core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -1507,7 +1507,8 @@ private void rewriteMerge(SqlMerge call) {
     // note that the values clause has already been converted to a
     // select on the values row constructor; so we need to extract
     // that via the from clause on the select
-    if (insertCall != null && config.sqlConformance().allowMergeInsertRewrite()) {
+    if (insertCall != null &&
```

Comment:
It looks like the checkstyle test fails unless "&&" is moved to the next line.

Reply:
Fix in https://github.com/googleinterns/calcite/pull/139/commits/eac9ee75ca51ce9dbc1efd141b1bfbe8ecb98e1a